servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/DefaultExecutor.java

```diff
@@ -249,11 +254,27 @@ public Cancellable schedule(final Runnable task, final long delay, final TimeUni
             // When using the global scheduler, offload timer ticks to the user specified Executor since user code
             // executed on the timer tick can block.
             ScheduledFuture<?> future = GLOBAL_SINGLE_THREADED_SCHEDULED_EXECUTOR.schedule(
-                    () -> offloadExecutor.execute(task), delay, unit);
-            // Schedulers are only used to generate a tick and do not execute any user code. This means they will never
-            // run any blocking code and hence it does not matter whether we use the interruptOnCancel as sent by the
-            // user upon creation in the scheduler. User code (completion of Completable on tick) will be executed on
-            // the configured executor and not the Scheduler thread.
+                    () -> {
+                        try {
+                            offloadExecutor.execute(task);
+                        } catch (Throwable t) {
```

Comment:
I wonder if the attempt to re-execute the task should only happen for `RejectedExecutionException` (ignoring `NullPointerException` for null task) as all other cases should be presumed to be caused by the task and re-running it would either cause the same exception or other problems.

Reply:
We should still log any `Throwable`. Otherwise, it's a hidden error with zero visibility. 
But I'm agreed, that we should run the `Runnable` only for `RejectedExecutionException`.